### PR TITLE
Fix/Align API names -- making V2 default and cleaning up matrix equations

### PR DESCRIPTION
**Pull request by alheinecke**

### CODE REVIEW

**Comment by alheinecke** [comment]
@egeor : several equation examples were still using the old APIs. They are now broken. Can yon please help with porting? Also do you have a script for this?

**Comment by egeor** [comment]
> @egeor : several equation examples were still using the old APIs. They are now broken. Can yon please help with porting? Also do you have a script for this?

Nope, all this is done manually...

**Comment by alheinecke** [comment]
@egeor  please review....

@ddkalamk @ska278 : in the moment @egeor approves I will merge this PR and #848

File: documentation/libxsmm_compat.md
Comment (egeor) [inline documentation/libxsmm_compat.md:1]: The search/replace of v2 seemed to mess up some docs

Comment (alheinecke) [reply documentation/libxsmm_compat.md:1]: fixed

**Review by egeor — APPROVED** [review]

**Pull request merged**